### PR TITLE
Fix max width of keyboard element

### PR DESCRIPTION
**Pull request by yanfali**
@noroadsleft I think this is probably the way we should go for now.

- also cleaned up CSS technical debt using Violet Pena's recommendations
  @see https://vgpena.github.io/using-css-grid-the-right-way/

### CODE REVIEW

File: src/components/ControllerTop.vue [hunk 1]
```diff
@@ -309,7 +309,7 @@ export default {
   data: () => {
     return {
       keymapName: '',
-      width: 0
+      width: 'max-width: 18rem'
```
Comment (noroadsleft) [inline src/components/ControllerTop.vue]: This is the only part of this change I don't understand how it works.

\- @noroadsleft 
*newb who doesn't understand the dark magic that is Vue*

Comment (yanfali) [reply src/components/ControllerTop.vue]: yeah, it's basically a component value that is templated into the page. We can move it into the css. I think the original intention was the size of the element was dynamically adjusted based on the width of the text. I got rid of that feature but didn't remove the templating.

Comment (yanfali) [reply src/components/ControllerTop.vue]: I'm going to use the CSS you proposed

Comment (noroadsleft) [reply src/components/ControllerTop.vue]: We can revisit this method later on when I understand wtf is going on here.

Comment (yanfali) [reply src/components/ControllerTop.vue]: oh yeah for sure. I actually used your css and removed the code.

**Comment by yanfali** [comment]
Verbal agreement from @noroadsleft

**Pull request merged**